### PR TITLE
Project settings update

### PR DESCRIPTION
**Pull request by sravfeyn**
[Spec](https://docs.google.com/a/dimagi.com/document/d/1W1WFzM_l1U-5VDH8MOfG22YqhlFbPz94Ttsl1H_2Q08/edit#heading=h.q3wyha58n4s0) for this
This PR renames/modifies/moves/deletes settings on different Project Settings pages to make it more minimal/structured. This doesn't delete attributes on any model yet. Will clean the models in a separate PR once these UI changes are live for a while on the prod.

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by sravfeyn** [comment]
I have made commits for almost every setting, so that it's easy to add anything back

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by snopoke** [comment]
Code looks fine.

**Comment by biyeun** [comment]
Looks good

**Comment by sravfeyn** [comment]
Thanks simon, biyeun, Triggered UAT, hey @christineawhite can you update this after finishing UAT http://manage.dimagi.com/default.asp?155353#863465

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/domain/forms.py [hunk 1]
```diff
@@ -632,68 +516,146 @@ def save(self, request, domain):
...
+            apps_to_save = []
+            if secure_submissions != domain.secure_submissions:
+                for app in get_apps_in_domain(domain.name):
+                    if app.secure_submissions != secure_submissions:
+                        app.secure_submissions = secure_submissions
+                        apps_to_save.append(app)
+            domain.secure_submissions = secure_submissions
+            domain.save()
+            if apps_to_save:
+                ApplicationBase.bulk_save(apps_to_save)
```
Comment (dimagimon) [inline corehq/apps/domain/forms.py]: F821 undefined name 'ApplicationBase'

**Comment by christineawhite** [comment]
@sravfeyn is this ready for QA again?

**Comment by sravfeyn** [comment]
Yeah!

**Comment by christineawhite** [comment]
@sravfeyn this passed QA, good to merge

**Comment by sravfeyn** [comment]
Thanks Christy

File: corehq/apps/domain/forms.py (hunk 1 above)
Comment (snopoke) [inline corehq/apps/domain/forms.py]: This needs to be done like this: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/forms.py#L485-485

**Comment by snopoke** [comment]
@sravfeyn good to merge once you fix the `ApplicationBase.bulk_save` thing

File: corehq/apps/domain/forms.py [hunk 2]
```diff
@@ -632,68 +517,153 @@ def save(self, request, domain):
...
+                for app in get_apps_in_domain(domain.name):
+                    if app.secure_submissions != secure_submissions:
+                        app.secure_submissions = secure_submissions
+                        apps_to_save.append(app)
+            domain.secure_submissions = secure_submissions
+            domain.save()
+
+            if apps_to_save:
+                apps = [app for app in apps_to_save if isinstance(app, Application)]
+                remote_apps = [app for app in apps_to_save if isinstance(app, RemoteApp)]
```
Comment (dimagimon) [inline corehq/apps/domain/forms.py]: F821 undefined name 'RemoteApp'

File: corehq/apps/domain/forms.py [hunk 3]
```diff
@@ -632,68 +517,153 @@ def save(self, request, domain):
...
+                if apps:
+                    Application.bulk_save(apps)
+                if remote_apps:
+                    RemoteApp.bulk_save(remote_apps)
```
Comment (dimagimon) [inline corehq/apps/domain/forms.py]: F821 undefined name 'RemoteApp'

File: corehq/apps/domain/forms.py [hunk 4]
```diff
@@ -36,7 +36,8 @@
     SubscriptionAdjustmentMethod,
     SubscriptionType,
 )
-from corehq.apps.app_manager.models import Application, FormBase, RemoteApp, get_apps_in_domain
+from corehq.apps.app_manager.models import (Application, RemoteApp
+                                            FormBase, get_apps_in_domain)
```
Comment (dimagimon) [inline corehq/apps/domain/forms.py]: E901 SyntaxError: invalid syntax

Comment (sravfeyn) [inline corehq/apps/domain/forms.py]: sorry, ran history command accidentally

**Comment by czue** [comment]
nice! been appreciating this change on staging for a while now...

**Pull request merged**